xgcm/grid.py

```diff
@@ -1361,24 +1361,42 @@ def iterate_axis_combinations(items):
         # return the product of the metrics
         return functools.reduce(operator.mul, metric_vars, 1)
 
-    def interp_like(self, da_source, metric):
+    @docstrings.dedent
+    def interp_like(self, da_target, da_source):
+        """Compares positions between two data arrays and interpolates missing values
+
+        Parameters
+        ----------
+        da_target : DatArray
+            Original array for comparison with target array
+        da_source : DataArray
+            Specifies which positions and coordinates should be present in the target array
+
+        Returns
+        -------
+        da_source : DataArray
+            Source data array with updated positions along axes compared with the target array
+        """
+
         for axname, axis in self.axes.items():
+            # This will raise a KeyError since this for-loop goes through all axes contained in self,
+            # but it is possible to apply the method for only 1 axis at a time
             try:
-                position_da, _ = axis._get_axis_coord(da_source)
-                position_metric, _ = axis._get_axis_coord(metric)
+                position_da_target, _ = axis._get_axis_coord(da_target)
+                position_da_source, _ = axis._get_axis_coord(da_source)
             except KeyError:
```

Comment:
Can you add a little comment for other contributors, why you expect a key error?